### ISSUE TITLE
Training Condition

### ISSUE THREAD

**Issue by reminayano** [YapengTian/TDAN-VSR-CVPR-2020#49]
I created a scale x2 model with the following conditions, but I could not get a better PSNR than bicubic. How did you set the epochs, learning rate, etc. when creating the scale 4 model?

---
train data (vimeo seplet dataset)
   HR : 448×256
   LR : 224x128
scale : 2
batchsize : 64
leaning rate : 1e-4
num_epochs : 600

**Comment by YapengTian** [comment]
Sorry for the very late response. Did you change the scale to 2 in both the train and dataloader code?  I think the params are ok.

**Comment by reminayano** [comment]
Thank you for your response.

Yes, I changed scale at train.py and SR_datasets.py(class VSR_Dataset).
